jwst/datamodels/model_base.py

```diff
@@ -253,12 +253,29 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.close()
 
+    def _drop_arrays(self):
+        def _drop_array(d, p):
```

Comment:
Why the extra argument for a locally defined function that is not used? Suggest removal.